runtime/shared_common/CacheMap.cpp

```diff
@@ -4303,18 +4300,44 @@ SH_CacheMap::releasePrivateSharedData(J9VMThread* currentThread, const J9SharedD
 }
 
 /* THREADING: Can be called at any time by any thread. Should not try to get any locks as it may be being
- * called as a result of a deadlock. The only locks obtained are by the managers when querying their hashtables. */
+ * called as a result of a deadlock. The only locks obtained are by the managers when querying their hashtables.
+ * 
+ * Get the J9SharedClassJavacoreDataDescriptor from the cache of all layers.
+ * 
+ * @param[in] vm  The J9JavaVM
+ * @param[out] descriptor The J9SharedClassJavacoreDataDescriptor
+ * 
+ * @return 1 on success and 0 otherwise
+ * */
 UDATA 
 SH_CacheMap::getJavacoreData(J9JavaVM *vm, J9SharedClassJavacoreDataDescriptor* descriptor)
+{
+	return getJavacoreData(vm, descriptor, false);
+}
+
+/* THREADING: Can be called at any time by any thread. Should not try to get any locks as it may be being
+ * called as a result of a deadlock. The only locks obtained are by the managers when querying their hashtables. 
+ * 
+ * Get the J9SharedClassJavacoreDataDescriptor for the current shared cache.
+ * 
+ * @param[in] vm  The J9JavaVM
+ * @param[out] descriptor The J9SharedClassJavacoreDataDescriptor
+ * @param[in] topLayerOnly  Whether J9SharedClassJavacoreDataDescriptor the from the cache of top layer only or all layers
```

Comment:
from the top layer cache only or all layers

Reply:
Fixed